CountryPicker/CountryPicker/CountryPickerWithSectionViewController.swift

```diff
@@ -72,6 +73,8 @@ internal extension CountryPickerWithSectionViewController {
     
     func scrollToCountry(_ country: Country, withSection sectionTitle: Character, animated: Bool = false) {
         
+        if applySearch { return }
+        
```

Comment:
Is this was the cause of exception (crash) ?

Reply:
Yes, when there's a previously selected country loaded then proceed with searching. `scrollToCountry` function gets to be invoked with incorrect row index to update the table view.